traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingResultMapListController.swift

```diff
@@ -15,21 +15,60 @@ public protocol ParkingResultMapListViewControllerDelegate: class {
     func parkingResultMapListViewController(_ controller: ParkingResultMapListViewController, didFinishWith purchaseForm: PurchaseForm)
 }
 
+let searchButtonBottomSpacing: CGFloat = 13
```

Comment:
Dont put variables in the global scope. This makes them available to all entities in the code and therefore makes it unsafe. All these variables are specific to this view controller and they belong to that class scope.